src/PuliExtension.php

```diff
@@ -99,7 +100,7 @@ public function getTokenParsers()
     public function getFunctions()
     {
         if (!$this->urlGenerator) {
-            return array();
+            throw new RuntimeException('The resource_url() Twig function requires a UrlGenerator to be set in PuliExtension.');
```

Comment:
If I remove this part (urlGenerator test), a RuntimeException is thrown by the [TemplatePathResolver::processResourceUrlFunction](https://github.com/puli/twig-extension/blob/master/src/NodeVisitor/TemplatePathResolver.php#L154:L159).
The `TemplatePathResolver` exception is only thrown when the `resource_url`is used.
Must we add an exception ?